components/Statistics/OrgStatisticsTabs.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react'
+import { Tabs } from 'antd'
+import VTabs from '../VTheme/VTabs'
+import { connect } from 'react-redux'
+import { withOrgs } from '../../lib/redux/reduxApi.js'
+
+import ReduxLoading from '../Loading'
+import StatisticsPanel from './StatisticsPanel'
+
+export const OrgStatisticsTabs = (props) => {
+  useEffect(() => {
+    props.dispatch(
```

Comment:
consider switching to useDispatch and useSelector instead of mapStateToProps can connect - its clearer and more hooky